src/data/global.json

```
@@ -277,6 +285,14 @@
                         {
                             "title": "FAQ",
                             "url": "/de/faq/"
+                        },
+                        {
+                            "id": "twitter",
+                            "icon": "twitter",
+                            "icon-url": "/assets/img/icons/twitter_white.png",
+                            "title": "Das Corona-Warn-App-Project auf Twitter",
```

Was it intended to use "Project" in the German version too?

```suggestion
                            "title": "Das Corona-Warn-App-Projekt auf Twitter",
```